README.md

```diff
@@ -176,8 +164,18 @@ This set of files contains the main results which includes the following:
 3. **rFon12**:  This is the regression of the trait onto *pairs* of sites for given nucleotides at each site. These are regressions on (site 1)x(site 2) independent of first order associations. Since we're looking at 2 sites at a time and there's a possibility of having 4 nucleotides at each site (for the case of DNA), we can visualize this via a 4x4 matrix as shown in Figure 8 in the paper linked above.
 
 
+# The rf1d class
+The newest update to *ortho_seqs* involves adding a new class of objects, called *rf1d* (short for rFon1D). To declare an *rf1d* object, you must supply a numpy ndarray of the rFon1D values (supplied via the regressions.npz file that is output when *ortho_seqs* is run), along with the alphabet input, and the molecule type.
+Once you have done this, you can explore the rFon1D values, and see if there are any trends. You can:
```

Comment:
can you show the user how they can actually do this? A quick example maybe on the command line? Otherwise, we could make this a command (like orthogonal_polynomial is and like gui is (see gui_draft branch)). In this case, once the user has their regressions.npz file, they can do something like: 
`ortho_seq rf1d_plot_orwhateverwecallthiscommand regressions.npz --molecule protein --alphbt_input G,S,Y,R`

Reply:
I see what you're saying. I have no experience with this, so it might take a few to get it set up, but that sounds like a good idea!

Reply:
I'm not quite sure how this would work, since the functions are all very different, this might have to be a future thing, since I'd have to work out some logicistical things with this